web/modules/custom/parser/js/parser.js

```diff
@@ -0,0 +1,23 @@
+window.onload = function() {
+  if (window.jQuery) {
+    jQuery("#edit-actions-submit").click(function() {
+      var checkBoxes = document.getElementsByClassName("form-checkbox");
+      var message = [];
+
+      Array.prototype.forEach.call(checkBoxes, (cb) => {
+        if (cb.checked) {
+          message.push(" " + cb.id);
+        }
+      })
+
+      if (message != "") {
+        if (!confirm("Clearing the tables of:" + message + "\nAre you sure?")) {
+          event.preventDefault();
+        }
+      }
+    });
+  } else {
+    alert("Your browser does not support jQuery doesn't Work");
```

Comment:
can we rephrase this alert? I feel like it doesn't give useful info, the user can be like "what doesn't work? what happened with the data? what should I do?"